sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-configcenter/src/main/java/org/apache/shardingsphere/orchestration/core/configcenter/ConfigCenter.java

```diff
@@ -180,13 +180,12 @@ private void persistShardingSchemaName(final String shardingSchemaName) {
             repository.persist(node.getSchemaPath(), shardingSchemaName);
             return;
         }
-        List<String> schemaNameList = Splitter.on(",").splitToList(shardingSchemaNames);
+        List<String> schemaNameList = Lists.newArrayList(Splitter.on(",").split(shardingSchemaNames));
```

Comment:
why?

Reply:
splitToList return readOnly List,and Then the list will add,